core/rawdb/freezer_table.go

```diff
@@ -120,7 +121,7 @@ type freezerTable struct {
 
 // NewFreezerTable opens the given path as a freezer table.
 func NewFreezerTable(path, name string, disableSnappy bool) (*freezerTable, error) {
-	return newTable(path, name, metrics.NilMeter{}, metrics.NilMeter{}, metrics.NilGauge{}, freezerTableSize, disableSnappy)
+	return newTable(path, name, metrics.NilMeter{}, metrics.NilMeter{}, metrics.NilGauge{}, freezerTableSize, disableSnappy, false)
```

Comment:
This function is used by `freezerInspect` which should open freezer in read-only mode.

But let's add one more parameter `readOnly` in this function but just blindly use `false` here.